src/test/kotlin/org/rustSlowTests/RsCargoCheckAnnotatorTest.kt

```diff
@@ -131,6 +133,25 @@ class RsCargoCheckAnnotatorTest : RsWithToolchainTestBase() {
         }
     }
 
+    fun `test strip document later reflection`() {
```

Comment:
If I understand correctly, this test checks that reflection doesn't throw exceptions
If yes, let's add comment about it